### PR TITLE
Add new pages for gh-pages

### PR DESCRIPTION
**Pull request by kevindougherty-noaa**
Now that website from GH-Pages is working, added sidebar with more pages for further instructions on how to use PyGSI.

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+The Github repository can be found here: https://github.com/NOAA-EMC/PyGSI
+and can be cloned by using the follow command:
+
+```
+$ git clone https://github.com/NOAA-EMC/PyGSI.git
+```
+This will clone the develop branch into a directory called PyGSI.
+
+## Setting Environment
+
```
Comment (CoryMartin-NOAA) [inline docs/getting_started.md:14]: I would add some sort of language here that is like:
```
Use on supported platforms
```

and then below it, instructions for installation into a local copy or other machine using pip

Comment (CoryMartin-NOAA) [reply docs/getting_started.md:14]: sorry, below this section with Hera/Orion, have a new section with installation instructions

File: docs/using_pygsi.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
+# Using EMCPy
+
+## Using `diags.py`
+
+[`diags.py`](https://github.com/NOAA-EMC/PyGSI/blob/develop/src/pyGSI/diags.py) is the mian driver script for extracting data from GSI diagnostic netCDF4 files. It performs all of the heavy lifting to read, filter, and perform quality control of the data so the user does not have to do this themselves. Given the data type and a diagnostic file, `diags.py` will return the user a [Pandas](https://pandas.pydata.org/docs/index.html) dataframe they can use to perform their analysis.
```
Comment (CoryMartin-NOAA) [inline docs/using_pygsi.md]: mian -> main

**Review by EdwardSafford-NOAA — APPROVED** [review]
Looks good to me.  I didn't spot any typos and the organization makes sense.

**Review by CoryMartin-NOAA — APPROVED** [review]

**Pull request merged**